android/KMAPro/kMAPro/src/main/java/com/tavultesoft/kmapro/KeymanSettingsFragment.java

```diff
@@ -56,7 +57,47 @@ as part of the default onClick() used by SwitchPreference.
     getStartedPreference.setTitle(getString(R.string.show_get_started));
     getStartedPreference.setDefaultValue(true);
 
+    setSystemKeyboardPreference = new CheckBoxPreference(context);
+    setSystemKeyboardPreference.setTitle(R.string.enable_system_keyboard);
+    setSystemKeyboardPreference.setDefaultValue(GetStartedActivity.isEnabledAsSystemKB(context));
+    setSystemKeyboardPreference.setOnPreferenceClickListener(new Preference.OnPreferenceClickListener() {
+      @Override
+      public boolean onPreferenceClick(Preference preference) {
+        startActivity(new Intent(Settings.ACTION_INPUT_METHOD_SETTINGS));
+        return false;
+      }
+    });
+    setSystemKeyboardPreference.setOnPreferenceChangeListener(new Preference.OnPreferenceChangeListener() {
+      @Override
+      public boolean onPreferenceChange(Preference preference, Object newValue) {
+        // Blocks the checkmark from changing from user interaction.
```

Comment:
This comment confused me for a while. But now I think I understand what it means -- the checkbox only gets updated when the user changes the setting in the activity that is triggered by touching the preference item. Can we clarify that a little?